### ISSUE TITLE
replaced by github.com/vmihailenco/encoding@v0.3.4-0.20220121071420-f96fbbb25975 - invalid: unknown revision f96fbbb25975

### ISSUE THREAD

**Issue by tkoeppen** [uptrace/uptrace#390]
when cloning the project first time and try to download dependencies
(environment: go version go1.22.4 darwin/arm64)
I get this error:

```
git clone https://github.com/uptrace/uptrace
cd uptrace
git submodule update --init --recursive
go clean -modcache
make gomoddownload go mod download
go: github.com/segmentio/encoding@v0.4.0 (replaced by github.com/vmihailenco/encoding@v0.3.4-0.20220121071420-f96fbbb25975): version "v0.3.4-0.20220121071420-f96fbbb25975" invalid: unknown revision f96fbbb25975
make: *** [gomoddownload] Error 1
```

https://github.com/vmihailenco/encoding

**Comment by tkoeppen** [comment]
look like v0.3.4 is not published, last tag is v0.3.3
https://github.com/vmihailenco/encoding/tags

**Comment by vmihailenco** [comment]
Hopefully should be fixed by https://github.com/uptrace/uptrace/commit/8ed72ec878dddbc7331cc19d355ba908a94fd652
